packages/ovee/src/utils/registerCustomElement.ts

```diff
@@ -1,8 +1,11 @@
-// eslint-disable-next-line no-unused-vars
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export default function (component: CustomElementConstructor, name: string, extendsEl?: string): void {
-    if (window.customElements) {
-        window.customElements.define(name, component);
-        // window.customElements.define(name, component, { extends: extendsEl });
-    }
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export default function(
+	component: CustomElementConstructor,
+	name: string,
+	extendsEl?: string
+): void {
+	if (window.customElements) {
+		window.customElements.define(name, component);
+		// window.customElements.define(name, component, { extends: extendsEl });
```

Comment:
would be good to further investigate, why `extends` refuses to work

Reply:
I moved it to a seperate issue #19